akvo/rest/views/typeahead.py

```diff
@@ -96,7 +96,7 @@ def typeahead_project(request):
     if request.GET.get('published', '0') == '0':
         # Project editor - organization projects, all
         page = request.rsr_page
-        projects = page.organisation.all_projects() if page else Project.objects.all()
+        projects = page.all_projects() if page else Project.objects.all()
```

Comment:
If we are showing unpublished projects also, I suspect private projects are fine. 
May be it doesn't make sense to add unpublished/private projects as related projects (that's where this end-point seems to be used).